libraries/pbrlib/genmdl/pbrlib_genmdl_impl.mtlx

```diff
@@ -23,7 +23,7 @@
   <implementation name="IM_subsurface_bsdf_genmdl" nodedef="ND_subsurface_bsdf" sourcecode="materialx::pbrlib_{{MDL_VERSION_SUFFIX}}::mx_subsurface_bsdf(mxp_weight:{{weight}}, mxp_color:{{color}}, mxp_radius:{{radius}}, mxp_anisotropy:{{anisotropy}}, mxp_normal:{{normal}})" target="genmdl" />
 
   <!-- <sheen_bsdf> -->
-  <implementation name="IM_sheen_bsdf_genmdl" nodedef="ND_sheen_bsdf" sourcecode="materialx::pbrlib_{{MDL_VERSION_SUFFIX}}::mx_sheen_bsdf(mxp_weight:{{weight}}, mxp_color:{{color}}, mxp_roughness:{{roughness}}, mxp_normal:{{normal}}, mxp_base:{{base}})" target="genmdl" />
+  <implementation name="IM_sheen_bsdf_genmdl" nodedef="ND_sheen_bsdf" sourcecode="materialx::pbrlib_{{MDL_VERSION_SUFFIX}}::mx_sheen_bsdf(mxp_weight:{{weight}}, mxp_color:{{color}}, mxp_roughness:{{roughness}}, mxp_normal:{{normal}}, mxp_base:{{weight}})" target="genmdl" />
```

Comment:
Yes, otherwise one of the OpenPBR shader will fail compilation. If you look at the interface of [ND_sheen_bsdf](https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/pbrlib_defs.mtlx#L131) you will see there is no parameter called `base`. That fix should be forward ported to 1.39, alongside with the other change where the OpenPbr folder is added to the [list of GenMdl tests](https://github.com/AcademySoftwareFoundation/MaterialX/pull/2082/files#diff-4c127df51edb1a111feca2c15fc1d9e4a7aa5cc523b885d37e2bb426200808c6R360).

Reply:
That change is incorrect. Will remove.

Reply:
I will need to investigate why the MDL shadergen test is returning
```
D:\a\MaterialX\MaterialX\source\MaterialXTest\MaterialXGenShader\GenShaderUtil.cpp(486): warning:
  Could not find an input named 'base' on node 'fuzz_bsdf' in Aluminum_Brushed

D:\a\MaterialX\MaterialX\source\MaterialXTest\MaterialXGenShader\GenShaderUtil.cpp(489): FAILED:
  CHECK( shader )
with expansion:
  00000000

D:\a\MaterialX\MaterialX\source\MaterialXTest\MaterialXGenShader\GenShaderUtil.cpp(893): FAILED:
  CHECK( codeGenerationFailures == 0 )
with expansion:
  1 == 0
```
For all OpenPBR example files.

Reply:
OK, found the cause and fixed it in https://github.com/AcademySoftwareFoundation/MaterialX/pull/2082/commits/955ed349c937787e9125a3e03ef34b6db62a8335. This prevents the MDL backport from being split into a separate library since this specific backport requires C++ shadergen code updates.